### DISCUSSION TITLE
Broken link in throttling documentation section ("how the `X-Forwarded-For` header works")

### DISCUSSION THREAD

**Discussion by v-spassky** [encode/django-rest-framework#9513] [Potential Issue] [answered]
On [this](https://www.django-rest-framework.org/api-guide/throttling/#how-clients-are-identified) page, the link shown below is broken:

![image](https://github.com/user-attachments/assets/fad725fc-158b-4fb8-9285-3c642c1161b0)

![image](https://github.com/user-attachments/assets/5686d3cc-10b2-45b9-b4ea-e52112ab24b4)

Is it supposed to point here: https://oxpedia.org/wiki/index.php?title=AppSuite:Grizzly#X-FORWARDED-FOR_Header ?

**Comment by lovelydinosaur** [comment]
Mozilla have historically been pretty reliable… perhaps we could point here… https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For#selecting_an_ip_address

Would you be up for issuing a PR resolving this?

**Comment by browniebroke** [comment, marked as answer]
This link works and the page loads fine for me now. It looks like this:

<img width="1684" alt="image" src="https://github.com/user-attachments/assets/999f6cdc-73b4-49b7-807a-b104f9c9eccc">

The diagram does a good job at explaining why there are multiple IPs in the header, but on the other hand it seems that Mozzila would be more trustworthy (I've never heard of OXpedia before). Perhaps let's leave it as is for now (if ain't broke, don't fix it)?
